tests/test_outlier.py

```diff
@@ -270,6 +270,16 @@ def test_class_public_func():
         OOD_ood_already_fit.params["confident_thresholds"] == confident_thresholds
     ).all()  # Assert not overwritten
 
+    # Testing fit with correct metrics given feature-space size
+    X_small = np.random.rand(20, 3)
+    OOD_euclidean = OutOfDistribution()
+    OOD_euclidean.fit(features=X_small)
+    assert OOD_euclidean.params["knn"].metric == "euclidean"
+    X_large = np.random.rand(20, 4)
+    OOD_cosine = OutOfDistribution()
+    OOD_cosine.fit(features=X_large)
+    assert OOD_cosine.params["knn"].metric == "cosine"
+
```

Comment:
Can you also assert the scores between 0 and 1 under either metric? 

And also if you add one outlier, that its score is closer to 0 than any of the other datapoints' scores?